### DISCUSSION TITLE
How to bulk create and connect?

### DISCUSSION THREAD

**Discussion by tusharkhatiwada** [prisma/prisma#10742] [Q&A]
Hi, I am trying to do and learn backend stuff. I came across prisma and tried to use it. I am having trouble with a relational query.

A Student Group can have multiple Students. 
A Student can be in multiple student groups. 

I need to import students from excelsheet and assign them to a group. I created a schema like this:

```prisma
model Student {
  id            String             @id @default(cuid())
  studentId     String             @unique
  firstName     String
  lastName      String
  email         String             @unique
  createdAt     DateTime           @default(now())
  updatedAt     DateTime           @updatedAt
  userId        String
  user          User               @relation(fields: [userId], references: [id], onDelete: Cascade)
  studentGroups StudentWithGroup[]
}

model StudentGroup {
  id         String             @id @default(cuid())
  userId     String
  name       String
  activeFrom DateTime
  activeTo   DateTime
  createdAt  DateTime           @default(now())
  updatedAt  DateTime           @updatedAt
  collegeId  String
  college    College            @relation(fields: [collegeId], references: [id], onDelete: Cascade)
  user       User               @relation(fields: [userId], references: [id], onDelete: Cascade)
  students   StudentWithGroup[]
}

model StudentWithGroup {
  studentId String
  groupId   String
  createdAt DateTime     @default(now())
  updatedAt DateTime     @updatedAt
  group     StudentGroup @relation(fields: [groupId], references: [id], onDelete: Cascade)
  student   Student      @relation(fields: [studentId], references: [id], onDelete: Cascade)

  @@id([studentId, groupId])
}
```
This is the query I am using, butt seems like it's not working:

```ts
prisma.student.createMany({
        data: data.map((student: any) => ({
          ...student,
          userId: id,
          studentGroups: { connect: { id: groupId } },
        })),
      });
```
This is the error I am getting:

```
**Unknown arg `studentGroups` in data.0.studentGroups for type StudentCreateManyInput. Available args:**

type StudentCreateManyInput {
  id?: String
  studentId: String
  firstName: String
  lastName: String
  email: String
  createdAt?: DateTime
  updatedAt?: DateTime
  userId: String
}
```
Any help will be much appreciated. 

Thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
